### ISSUE TITLE
Don't reference the full aws-sdk

### ISSUE THREAD

**Issue by lukestephenson** [mboudreau/Alternator#98]
Adding this library to a project downloads the full suite of aws java libraries.  Most are irrelevant (eg beanstalk, elasticcache, cloudfront, s3 etc).

The pom should list only the required dependencies.

Thoughts?  Do people agree?

(btw I came across this project as getting amazons local dynamo db running in a unit test environment is a pain - great work!)

**Comment by rrutt** [comment]
As Amazon adds more services, and this expands their SDK, this becomes a more relevant issue.

Do we know what the pom.xml dependency artifact values are for the DynamoDB, and perhaps the authentication, modules from AWS SDK?

**Comment by mboudreau** [comment]
@lukestephenson  yep, I'd agree with that. I haven't touched the project in a while, but if you want to create the PR, I'll merge and release the new version.
